Yoast/YoastSEO.js
# Fix sentence breaker matching

## Summary

This PR can be summarized in the following changelog entry: Fixes a bug where the passive voice detection was broken because of incorrect sentence breaker matching.

## Test instructions

This PR can be tested by following these steps:

- Use the browserified example. Don't forget to run a `grunt build:js`.
- Add `console.log(sentencePart)` to `js/researches/english/determinePassives`
- Write `Who commented is praise due`.
- The logged sentence part is `is pra`

- Check out this branch.
- Don't forget to run a `grunt build:js`.
- Add the console.log for the sentencePart.
- The logged sentence part should be `is praise due`.

Fixes #1210
